github-pages/Dockerfile

```
@@ -1,24 +1,20 @@
 # https://pages.github.com/versions/
-FROM ruby:2.5-slim
+FROM ruby:2.6-alpine
```

Given that https://pages.github.com/versions/ still lists Ruby 2.5, I'd prefer to stick to 2.5 here as well, but moving to Alpine seems sane as long as we pin it while we're here:

```suggestion
FROM ruby:2.5-alpine3.10
```

---

See https://github.com/github/pages-gem/pull/647

---

https://rubygems.org/gems/github-pages just mention `>= 2.3.0`

---

Yeah, that's reasonable, but my goal with this image was to be able to test my GitHub pages deployments locally, and for that IMO it makes more sense to mirror their production deployment more than what the Gem technically supports. :+1: